dirty_cat/_ken_embeddings.py

```diff
@@ -0,0 +1,92 @@
+"""
+Get the Wikipedia embeddings for feature augmentation.
+"""
+import pandas as pd
+from sklearn.decomposition import PCA
+
+from dirty_cat.datasets import fetch_embeddings
+
+
+def get_ken_embeddings(
+    types,
+    exclude=None,
+    pca_components=None,
+    emb_full_id="39142985",
+    emb_type_id="39143012",
+    suffix="",
+):
+    """Extract and clean Wikipedia embeddings by type.
+
+    More details on the embeddings can be found on
+    https://soda-inria.github.io/ken_embeddings/.
+
+    Parameters
+    ----------
+    types : str
+        Types of embeddings to include. Write in lowercase.
+    exclude : str, default=None
+        Type of embeddings to exclude from the types search.
+    pca_components : int, default=None
+        Size of the dimensional space on which the embeddings will be projected
+        by a principal component analysis.
+        If None, the default dimension (200) of the embeddins will be kept.
```

Comment:
Parameters emb_full_id and emb_type_id are not documented, and I'm not sure what they mean.